Kryptonite/AppDelegate.swift

```diff
@@ -23,7 +23,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
         Analytics.migrateOldIDIfExists()
         Analytics.migrateAnalyticsDisabled()
-                
+        
+        do {
+            try Caches.createCachesRootDirectory()
```

Comment:
can we just call this every time we ask for the cache directory? that way we don't need to worry about Notify trying to get a cache before AppDelegate is called